integration/utils.go

```diff
@@ -22,16 +22,8 @@ import (
 	"path"
 )
 
-func make(command, makefile string) error {
-	out, err := exec.Command("make", "-f", makefile, command).CombinedOutput()
-	if err != nil {
-		return fmt.Errorf("failed to build go binary (%q) - %q", err, out)
-	}
-	return nil
-}
-
 func buildDockerImage(imageName string) error {
-	out, err := exec.Command("docker", "build", "-t", imageName, ".").CombinedOutput()
+	out, err := exec.Command("./build.sh", imageName).CombinedOutput()
```

Comment:
Firstly, because both the script and this go func do essentially the same. So why duplicate code? For example, we found out that the bash code was wrong (it was missing the `go generate` stuff).

Secondly, because now that the root Makefile no longer uses absolute build paths, running make in the root dir will create a `heapster` binary in the root directory. So we'd have to copy it over. Which complicates the Go code in this function even more.

So it's more a question of whether to reimplement all of `build.sh` in Go, which seemed unnecessary.